contrib/debian/rules

```diff
@@ -125,7 +125,7 @@ override_dh_fixperms:
 	chmod 0750 $(TOP)-plugin-cups/usr/libexec/netdata/plugins.d/cups.plugin
 
 	# freeIPMI plugin package
-	chmod 0754 $(TOP)-plugin-freeipmi/usr/libexec/netdata/plugins.d/freeipmi.plugin
+	chmod 4750 $(TOP)-plugin-freeipmi/usr/libexec/netdata/plugins.d/freeipmi.plugin
```

Comment:
@MrFreezeex  Shouldn't this be 0750 just like the other plugins? And then change it to correct values in the postinst script.

Reply:
This is installed by the package netdata-plugin-freeipmi which do not "unset" (with chown) the suid in postinstall, so It shoud be sufficient...